raiden/tests/utils/tester_client.py

```diff
@@ -144,7 +144,10 @@ def closed_block(self):
 
     @property
     def settled_block(self):
-        return self.netting_channel.settled()
+        # XXX: Not sure about this. In normal operation (netting_channel.py)
+        # this is set from the settled event. Does this even make sense to
+        # be queried by tester?
+        return 0
```

Comment:
Nah, the tester should mirror the rpc client, since the smart contact doesn't have the attribute anymore we don't nee this.

Reply:
Hmm, so my comment here was that normally we have the `ChannelExternalState` which sets the `settled_block` when reading the events as [seen here](https://github.com/raiden-network/raiden/blob/f86c61debe56b320ed06e1962add4989e6d7ac00/raiden/channel/netting_channel.py#L91).

The class I edit here and for which this comment is about, `ChannelExternalStateTester`, seems to be the tester equivalent, which is why I would like to keep the same interface. Do I misunderstand something?

Reply:
Nope, I did though. I thought this was the rpc client equivalent.

This property doesn't make sense though, we should add the `set_settled` equivalent, otherwise someone may hit on weird test bugs in the future.

Reply:
This property is what queries the `set_settled()` result, so it has to stay. But we do indeed need to add the `set_settled()` equivalent. Made an issue for it: https://github.com/raiden-network/raiden/issues/1014